index.js

```diff
@@ -20,8 +23,49 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+   
+    Recipe.create({title:"Pizza", level: "Amateur Chef", cuisine: "Italian", ingredients: ["flour", 'cheese', 'tomato sauce', 'pure magic']})
+    .then((recipe) => {
+      console.log(recipe.title)
+    })
+    .catch()
+
+    Recipe.insertMany(data)
+    .then((allRecipes) =>{
+
+      allRecipes.forEach((recipe) => {
+        console.log(recipe.title)
+      })
+      let updatePromise = Recipe.updateOne({title:"Rigatoni alla Genovese"}, {duration:100})
+      updatePromise.then(()=>{
```

Comment:
the promise.all you are using later works as a way to check if the promises resolved or rejected. So doing a .then here might be redundant since you are checking that in the promise.all, same for the deletePromise one